### DISCUSSION TITLE
Wait time before cure

### DISCUSSION THREAD

**Discussion by MihailBuzea** [sn4k3/UVtools#626] [Q&A]
Hello all,

I have a small issue  regarding ''Wait time before cure'' settings,
I modify parameters bun noting actually applies to the print.
Also I try running the script made by Tiago Conceição, but as all we know is no longer possible to run this one

![wait time](https://user-images.githubusercontent.com/120726507/208104332-b8887d49-f289-4b0f-8872-01415f9e1bb6.PNG)

PS I am trying to get rid of "Elephant foot" i use a Anycubic Photon M3 Plus printer with a Formfutura  Strong black Resin that is very High Viscosity, and with Elephant foot calibration i have no visible improvement

**Comment by sn4k3** [comment]
None of the Anycubic machines are able to use this tool properly. They only offer 1 global wait time, which mean you are stuck with one wait time per all layers, due that UVtools set the normal value (4.50s) to all your layers, invalidating the porpouse of "eliminating" the EF with this tool. As the porpouse of this tool is also set wait times the best it can, it allow you to use it as long the file have any wait time to set.
